src/chain/network_event.rs

```diff
@@ -38,6 +39,14 @@ pub struct OnlinePayload {
     pub client_auth: Authority<XorName>,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
+pub struct AckMessagePayload {
+    /// The prefix acknowledging knowledge
```

Comment:
I thought it shall be consistent with the comment of `ack_version`,
i.e. the src_prefix means `the prefix acknowledged`?

Reply:
`the prefix acknowledged` for me would mean the prefix of the section info that ack_version is equal to. That would not be correct.
However, This is the prefix of our section when we acknowledge their section info that ack_version is equal to.

I'm happy to reword, but I want this distinction to be clear since that was the source of the bug.

Reply:
>This is the prefix of our section when we acknowledge their section info that ack_version is equal to

Ah, yes. This is much clear. Maybe we shall ues this as the comment.